library/system.lisp

```diff
@@ -102,9 +103,8 @@ While the result will always contain microseconds, some implementations may retu
      "The run time of the run" UFix)
```

Comment:
What are the units? Is UFix appropriate?

Reply:
They are microseconds, I think UFix makes sense?